File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/dashboard/data/MyDashboardResponse.java
package org.wikiedufoundation.wikiedudashboard.dashboard.data;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.ToString;

@Getter
@AllArgsConstructor
@ToString
public class MyDashboardResponse {
    private UserData user;
    private List<CourseListData> current_courses;
}

File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/dashboard/data/MyDashboardResponse.kt
package org.wikiedufoundation.wikiedudashboard.ui.dashboard.data

import com.google.gson.annotations.SerializedName
import org.wikiedufoundation.wikiedudashboard.ui.courselist.data.CourseListData

/**
 * [MyDashboardResponse] model class
 * @constructor primary constructor to initialize properties and variables
 * ***/
class MyDashboardResponse(
    @SerializedName("user")
    val user: UserData,
    @SerializedName("current_courses")
    val currentCourses: List<CourseListData>
)
